CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Manual stop recording button, on the 'recording' status icon.
```

Comment:
https://github.com/aza547/wow-recorder/issues/142 raised so you can link it to an issue.